zulipterminal/ui_tools/boxes.py

```diff
@@ -197,25 +197,25 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
                 self.view.set_footer_text()
```

Comment:
Re first commit: it would be useful to describe more of the reason for the refactor in the title and/or summary. That's made clearer in the second commit but just reading this commit it's less clear.